projects/plugins/boost/app/admin/class-admin.php

```diff
@@ -29,8 +29,7 @@ class Admin {
 	/**
 	 * Nonce action for setting the statuses of rating and score prompts.
 	 */
-	const SET_SHOW_RATING_PROMPT_NONCE = 'set_show_rating_prompt';
-	const SET_SHOW_SCORE_PROMPT_NONCE  = 'set_show_score_prompt';
+	const SET_SHOW_SCORE_PROMPT_NONCE = 'set_show_score_prompt';
 
```

Comment:
Just a heads up - a lot of this has been moved into a new `class-config.php` file in a [recent PR](https://github.com/Automattic/jetpack/pull/24808) we urgently squeezed through. If you need help with merging, let me know.